frontend/src/containers/__tests__/ChooseStaticDataset.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { fireEvent, render, screen, act } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import ChooseStaticDataset from "../ChooseStaticDataset";
+import BackendService from "../../services/BackendService";
+import userEvent from "@testing-library/user-event";
+
+jest.mock("../../hooks");
+jest.mock("../../services/BackendService");
+
+test("renders page title", () => {
+  render(<ChooseStaticDataset />, {
+    wrapper: MemoryRouter,
+  });
+  expect(
+    screen.getByRole("heading", { name: "Choose static dataset" })
+  ).toBeInTheDocument();
+});
+
+test("renders search box", () => {
+  render(<ChooseStaticDataset />, {
+    wrapper: MemoryRouter,
+  });
+  expect(screen.getByRole("searchbox", { name: "Search" })).toBeInTheDocument();
+});
+
+test("renders a selet and continue button", () => {
+  render(<ChooseStaticDataset />, {
+    wrapper: MemoryRouter,
+  });
+  expect(
+    screen.getByRole("button", { name: "Select and continue" })
+  ).toBeInTheDocument();
+});
+
+test("renders a cancel button", () => {
+  render(<ChooseStaticDataset />, {
+    wrapper: MemoryRouter,
+  });
```

Comment:
Nit pick: This render instruction is repeated on every test. You can extract it into a beforeEach: 

```js
beforeEach(() => {
  render(<ChooseStaticDataset />, {
    wrapper: MemoryRouter,
  });
});
```

Reply:
Adding another test to verify that a Table is rendered with the dummy datasets from your mocked hook would be good too.